elastic/apm-server
# Update to elastic/beats@49e8024953a4

##  Motivation/summary

Update beats to bring in https://github.com/elastic/beats/pull/20576.

## Checklist


I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

`PLATFORMS=linux/amd64 TYPES=docker mage package`

Should build a ubi8-minimal based image: `build/distributions/apm-server-ubi8-8.0.0-linux-amd64.docker.tar.gz`

## Related issues

https://github.com/elastic/apm-server/issues/4066
